### PR TITLE
Update control_plane.rst

### PR DESCRIPTION
**Pull request by SandraGH5**
# TL;DR
_Please replace this text with a description of what this PR accomplishes._

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by kumare3** [comment]
@SandraGH5 can we you add a better description please?

File: docs/source/design/control_plane.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ backend set up and have access to it (a :std:ref:`local sandbox <flyte:deploymen
 
 The video below features a demo of FlyteRemote, followed by all relevant information to create, fetch and execute FlyteRemote objects.
 
-..  youtube:: Vjukdg8LIh8
+..  youtube:: 1Dbj3t_1Ltg
```
Comment (kumare3) [inline docs/source/design/control_plane.rst:15]: this video seems to be private - https://www.youtube.com/watch?v=1Dbj3t_1Ltg

Comment (SandraGH5) [reply docs/source/design/control_plane.rst:15]: It is now public.

**Review by wild-endeavor — APPROVED** [review]

File: docs/source/design/control_plane.rst (hunk 1 above)
Comment (SandraGH5) [reply docs/source/design/control_plane.rst:15]: It is now public.

**Pull request merged**